packages/package.json

```diff
@@ -49,8 +49,8 @@
   },
   "peerDependencies": {
     "node-sass": ">= 7",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": ">=17.0.2",
+    "react-dom": ">=17.0.2",
```

Comment:
We don't want to block folks from upgrading Backpack in parallel to adopting React

Reply:
Should we limit it to `17.0.2 || ^18.3.1`? I don't think we're ready to tell that we support 19 as well, whenever it's released

Reply:
Ah yes good point! Completely forgot about that will update that now